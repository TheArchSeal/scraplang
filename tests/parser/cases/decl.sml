var x = 0;
const x = 0;
var x: u8 = 0;
