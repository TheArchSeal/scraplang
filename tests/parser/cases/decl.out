stmt (4):1:1 var x
    expr (2):1:9 0
    type (1):1:5 (inferred)
stmt (4):2:1 const x
    expr (2):2:11 0
    type (1):2:7 (inferred)
stmt (4):3:1 var x
    expr (2):3:13 0
    type (2):3:8 u8
